src/services/ui/src/features/package/package-actions/index.tsx

```diff
@@ -1,36 +1,37 @@
-import { useGetItem, useGetPackageActions } from "@/api";
-import { LoadingSpinner, Link } from "@/components";
+import { useGetItemCache } from "@/api";
+import { Link } from "@/components";
 import { mapActionLabel } from "@/utils";
-import { Action, Authority } from "shared-types";
 import { DetailCardWrapper } from "..";
 import { FC } from "react";
 import { useLocation } from "react-router-dom";
 
-export const PackageActionsCard: FC<{ id: string }> = ({ id }) => {
+import type { opensearch } from "shared-types";
+
+export const PackageActionsCard: FC<{
+  id: string;
+  data: opensearch.main.Document;
```

Comment:
we could rely solely on this prop instead of calling `useGetItemCache` below to reduce the weight of this component a little. I didn't see anywhere we used `item` over `data`